code/game/objects/items/bags.dm

```diff
@@ -146,3 +146,87 @@
 "sflip" = 0,
 "wflip" = 0,
 "eflip" = 8)
+
+/obj/item/storage/handbasket
+	name = "handbasket"
+	desc = "Fibers interwoven to make a cheap, handheld storage item."
+	icon_state = "handbasket"
+	icon = 'icons/roguetown/items/misc.dmi'
+	slot_flags = ITEM_SLOT_HIP
+	w_class = WEIGHT_CLASS_NORMAL
+	resistance_flags = NONE
+	max_integrity = 300
+	component_type = /datum/component/storage/concrete/grid/handbasket
+
+/obj/item/storage/handbasket/update_overlays()
+	. = ..()
+	var/list/stuff = contents
+	if(!length(stuff) >= 5)
```

Comment:
```suggestion
	if(length(stuff) < 5)
```
perhaps the ambiguious ! is why its always triggering? The idea is it would only get to that line if it had 5 of anything and not 5 of the lets say "whitelisted" items but this wasn't blocking it would cause it to happen when empty

Reply:
Ok I messed up